client/src/pages/OfferHelp.js

```diff
@@ -22,11 +22,15 @@ import {
   WizardCheckboxItem,
   WizardSubmit,
 } from "components/StepWizard";
+import createSendgridContact from "../utils/create-sendgrid-contact";
 
 const INITIAL_STATE = {
   answers: [],
 };
 
+// for now just creating an object to collect the data that I need
+let stepsData = {};
```

Comment:
Might think of having context instead of this obj, but for now this should work fine.